styles.css

```diff
@@ -1,33 +1,42 @@
 *{
   box-sizing:border-box;
 }
+
+header, footer{
+  flex: 0 0 100%;
+  background:#eee;
+}
+
+.main{
+  flex: 3;
+}
+
 body{
   margin:0;
   display: flex;
   flex-wrap: wrap;
   font-family
 }
-header, footer, aside, .main{
-  padding: 1em;
-}
-header, footer{
-  flex: 0 0 100%;
-  background:#eee;
-}
+
 aside{
   flex: 1;
   background:#ddd;
 }
-.main{
-  flex: 3;
+
+header, footer, aside, .main{
+  padding: 1em;
+  margin: 0
 }
```

Comment:
👍  to target direct children we'd add another rule with selector `header>*, footer>*, aside>*, .main>*`